README.md

```diff
@@ -1,9 +1,241 @@
-# Semantic Search for code - demo
+# Code search with Qdrant
 
-## Overview
+Developers need a code search tool that helps them find the right piece of code. In this README, we describe how
+you can set up a tool that provides code results, in context.
 
-ToDo
+## Online version
 
-## How to run
+See our code search tool "in action." Navigate to 
+**[https://code-search.qdrant.tech/](https://code-search.qdrant.tech/)**. We've prepopulated the demo with Qdrant 
+codebase. You can see the results, in context, even with relatively vague search terms.
 
-ToDo
\ No newline at end of file
+## Prerequisites
+
+To run this demo on your own system, install and/or set up the following components:
+
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/)
+- [Rust](https://www.rust-lang.org/learn/get-started)
+- [rust-analyzer](https://rust-analyzer.github.io/)
+
+Docker and Docker Compose setup depends on your operating system. Please refer to the official documentation for
+instructions on how to install them. Both Rust and rust-analyzer can be installed with the following commands:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+rustup component add rust-analyzer
+```
+
```

Comment:
```suggestion

If not already installed, you may need to install `rustup` first. In addition, make sure to install the `stable` version.

```

Reply:
Actually, the first line would install the `rustup` binary, so it should be available. Let me run it in a fresh environment.